osutil/isolate_win.go

```diff
@@ -10,4 +10,4 @@ import (
 // IsolateProcess change gpid for the process to avoid bypassing signals to php processes.
 func IsolateProcess(cmd *exec.Cmd) {
 	cmd.SysProcAttr = &syscall.SysProcAttr{CreationFlags: syscall.CREATE_NEW_PROCESS_GROUP}
-}
+}
```

Comment:
Don't you need to add a method here as well?

Reply:
We decided to drop windows support for that functionality.

Reply:
Done. Add a mock here for windows